.github/workflows/publish.yml

```diff
@@ -0,0 +1,78 @@
+name: publish
+on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: 'Dry run? (y/N)'
+        required: true
+        default: 'N'
```

Comment:
Generally, this workflow won't be run (we have the automated `release-please` below)

But it can be helpful if we needed to run dry-run, publish manually, or release a canary version